html/streamingtest.js

```diff
@@ -489,15 +491,6 @@ function addPanel(panelId, mid, desc) {
 	);
 }
 
-// Helper to escape XML tags
-function escapeXmlTags(value) {
-	if(value) {
-		var escapedValue = value.replace(new RegExp('<', 'g'), '&lt');
-		escapedValue = escapedValue.replace(new RegExp('>', 'g'), '&gt');
-		return escapedValue;
-	}
-}
-
```

Comment:
Why has this function been removed? I remember it was added in #2817 to avoid issues of user provided descriptions (e.g., names, titles, etc.) that could end up being interpreted wrongly or maliciously when used in web pages. Looking at the original PR, we definitely used it in the Streaming demo, so if we're not anymore I want to check why (maybe we're using `.text()` instead of `.html()`?).

Reply:
because it's being defined twice (https://github.com/meetecho/janus-gateway/blob/961b5764b34b5f84c56825f8acf05890d4551aa4/html/streamingtest.js#L466 and https://github.com/meetecho/janus-gateway/blob/961b5764b34b5f84c56825f8acf05890d4551aa4/html/streamingtest.js#L493)